.circleci/config.yml

```diff
@@ -35,7 +35,14 @@ jobs:
 
 workflows:
   version: 2
-  build_and_deploy:
+  commit_deploy:
```

Comment:
I think this workflow will still run on every push. And since there is only one job that is called from both workflows, the scheduled one will be _in addition_ to every push. Please correct me if I'm misunderstanding.